.github/workflows/github-actions-test.yml

```diff
@@ -0,0 +1,28 @@
+name: test
+on : [pull_request, push]
+jobs:
+  Explore-Github-Actions:
```

Comment:
Is this the name of the job? If possible, can we rename this?

Reply:
renamed to mcat-action-test!! Do you suggest any other name?

Reply:
that sounds good to me, thanks!